modules/@apostrophecms/util/ui/src/http.js

```diff
@@ -115,10 +115,10 @@ export default () => {
       return callback(new Error('url is not defined'));
     }
 
-    if (apos.prefix && options.prefix !== false) {
+    if (apos.sitePrefix && options.prefix !== false) {
```

Comment:
Not sure about the impact of this change.
We can have setups with 
* multiple locales but no prefix (DNS only)
* multiple locales some with prefix and some without
* multiple locales all with prefix

Can you check in all these contexts that the change is ok?

Reply:
The purpose of using `sitePrefix` is that it does not contain the locale in the prefix.
Otherwise it conflicts backend side with the one passed through `aposLocale`.